Classes/Emogrifier.php

```diff
@@ -304,6 +312,9 @@ protected function process(\DOMDocument $xmlDocument)
                     $oldStyleDeclarations = [];
                 }
                 $newStyleDeclarations = $this->parseCssDeclarationBlock($selector['attributes']);
+                if ($this->shouldMapCss2Html) {
+                    $this->mapStyles2HtmlAttrs($newStyleDeclarations, $node);
```

Comment:
Please let's keep the naming of the property and the method consistent: Either both should use the term "Styles" or "Css".